src/client/voice/VoiceConnection.js

```diff
@@ -457,7 +457,7 @@ class VoiceConnection extends EventEmitter {
    *   .catch(console.error);
    */
   playFile(file, options) {
-    return this.player.playUnknownStream(`file:${file}`, options);
+    return this.player.playUnknown(`file:${file}`, options);
```

Comment:
the file: prefix isn't needed, that was a stupid feature in old prism but that's gone now